### PR TITLE
Improve terminology on Scaling pages

### PR DESCRIPTION
**Pull request by kladkogex**
Changed "company" to "project" everywhere. There are no companies in decentralized world.

Mentioned that for rollups the operator can influence ordering.

Mentioned that sidechains similar to rollups look and feel as ETH main net

### CODE REVIEW

File: src/content/developers/docs/layer-2-scaling/index.md [hunk 1]
```diff
@@ -172,6 +172,7 @@ There are two types of channels right now:
 
 ## Plasma {#plasma}
 
+
```
Comment (wackerow) [inline src/content/developers/docs/layer-2-scaling/index.md:175]: can remove extra line

File: src/content/developers/docs/layer-2-scaling/index.md [hunk 2]
```diff
@@ -250,7 +254,7 @@ Combine the best parts of multiple layer 2 technologies, and may offer configura
 
 **ZK rollups**
 
-- [EthHub on zk-rollups](https://docs.ethhub.io/ethereum-roadmap/layer-2-scaling/zk-rollups/)
+- [EthHub on zk-rollups](https://docs.ethhub.io/ethereum-roadmap/layer-2-scaling/zk-rollups/)\
```
Comment (wackerow) [inline src/content/developers/docs/layer-2-scaling/index.md:257]: can remove this stray `\`

File: src/content/developers/docs/layer-2-scaling/index.md [hunk 3]
```diff
@@ -30,9 +30,9 @@ You should have a good understanding of all the foundational topics. Implementin
...
+Most layer 2 solutions are centered around a server or cluster of servers, each of which may be referred to as a node, validator, operator, sequencer, block producer, or similar term. Depending on the implementation, these layer 2 nodes may be run by the invividuals, businesses or entities that use them, or by a 3rd party operator, or by a large group of individuals (similar to mainnet). Generally speaking, transactions are submitted to these layer 2 nodes instead of being submitted directly to layer 1 ([mainnet](/glossary/#mainnet)). For some solutions the layer 2 instance then batches them into groups before anchoring them to layer 1, after which they are secured by layer 1 and cannot be altered. The details of how this is done vary significantly between different layer 2 technologies and implementations.
```
Comment (wackerow) [inline src/content/developers/docs/layer-2-scaling/index.md:33]: in**v**ividuals -> in**d**ividuals

**Review by wackerow — CHANGES_REQUESTED** [review]
@kladkogex Hey thanks for submitting =) Noted a couple small changes

LGTM, @ryancreatescopy let me know if you have any thoughts on the wording here

File: src/content/developers/docs/layer-2-scaling/index.md (hunk 2 above)
Comment (kladkogex) [reply src/content/developers/docs/layer-2-scaling/index.md:257]: OK

File: src/content/developers/docs/layer-2-scaling/index.md (hunk 3 above)
Comment (kladkogex) [reply src/content/developers/docs/layer-2-scaling/index.md:33]: OK :)

File: src/content/developers/docs/layer-2-scaling/index.md (hunk 1 above)
Comment (kladkogex) [reply src/content/developers/docs/layer-2-scaling/index.md:175]: Ok :) Thanks

**Comment by wackerow** [comment]
@kladkogex Just touching base, noticed you marked things as resolved but didn't see any commits... let me know if you're having any issues with GitHub or need any other help

**Review by jaceppo — APPROVED** [review]
guanlonghuang

**Comment by wackerow** [comment]
@kladkogex What's the status on this?

**Comment by minimalsm** [comment]
Cleaned this up as per your comments @wackerow but we have restructured this section into individual pages for each scaling solution. The merge conflicts require more wizardary than I feel comfortable attempting, is the easiest thing to do to open a new PR to make these changes in the corresponding, new pages?

**Review by wackerow — APPROVED** [review]
Well the merge/autoformatting/conflict clean up made a mess of this diff, but I simply merged the latest `dev` updates, removed the `/layer-2-scaling/` page and placed the changes into the `/scaling/`, `/scaling/layer-2-rollups/` and `/scaling/sidechains/`. 

Made very slight adjustments to the sidechain copy.

Should be good for a merge now after it builds 👍🏻

**Comment by minimalsm** [comment]
Awesome! Thanks @wackerow for sorting conflicts and thanks @kladkogex for the PR ✌️

Be sure to [join the discord](https://discord.gg/CetY6Y4) if you are interested in contributing further to the project or have any questions for the team. And remember to [claim your POAP](https://ethereum.org/en/contributing/#poap) also 🥳!

**Comment by minimalsm** [comment]
@all-contributors please add @kladkogex for docs

**Pull request merged**